pandas/tests/io/json/test_pandas.py

```diff
@@ -807,22 +810,24 @@ def test_convert_dates(self, datetime_series, datetime_frame):
         df = datetime_frame
         df["date"] = Timestamp("20130101").as_unit("ns")
 
-        json = StringIO(df.to_json())
+        json = StringIO(df.to_json(date_format="iso"))
         result = read_json(json)
         tm.assert_frame_equal(result, df)
 
         df["foo"] = 1.0
-        json = StringIO(df.to_json(date_unit="ns"))
+        json = StringIO(df.to_json(date_unit="ns", date_format="iso"))
 
         result = read_json(json, convert_dates=False)
         expected = df.copy()
-        expected["date"] = expected["date"].values.view("i8")
+        expected["date"] = expected["date"].apply(
```

Comment:
Similar comment here as above - I think should just keep the standard "epoch" behavior for now. Can update the expectation when epoch gets removed